hugegraph-api/src/main/java/org/apache/hugegraph/api/API.java

```diff
@@ -1,5 +1,4 @@
 /*
- * Copyright 2017 HugeGraph Authors
```

Comment:
can also remove the next blank line?

Reply:
@z7658329 I remove the extra line in commons & toolchains, you could update the model for server

and should check other files, like `Copyright 2019`, the regex settings shall be carefully

Reply:
> @z7658329 I remove the extra line in [commons](https://github.com/apache/incubator-hugegraph-commons/pull/124/commits/edc56b8e5f7b37e9819cd8a24b07ac1fa8e4e23c) & [toolchains](https://github.com/apache/incubator-hugegraph-toolchain/pull/421), you could update the model for server
> 
> and should check other files, like `Copyright 2019`, the regex settings shall be carefully

ok  i will check it